examples/tutorial/app/simple_wx.py issue

```
Traceback (most recent call last):
  File "simple_wx.py", line 58, in <module>
    frame = TestFrame()
  File "simple_wx.py", line 49, in __init__
    self.canvas = Canvas(app="wx", parent=self, show=True)
  File "simple_wx.py", line 20, in __init__
    app.Canvas.__init__(self, *args, **kwargs)
  File "/home/eldar/src/vispy/vispy/app/canvas.py", line 208, in __init__
    self.set_current()
  File "/home/eldar/src/vispy/vispy/app/canvas.py", line 406, in set_current
    self._backend._vispy_set_current()
  File "/home/eldar/src/vispy/vispy/app/backends/_wx.py", line 302, in _vispy_set_current
    self.SetCurrent(self._gl_context)
wx._core.wxAssertionError: C++ assertion "xid" failed at /home/eldar/src/wx/wxPython_Phoenix/wxPython-4.0.0b2/ext/wxWidgets/src/unix/glx11.cpp(194) in SetCurrent(): window must be shown
```

--- a/examples/tutorial/app/simple_wx.py
+++ b/examples/tutorial/app/simple_wx.py
@@ -47,12 +47,14 @@ class TestFrame(wx.Frame):
         MenuBar.Append(file_menu, "&File")
         self.SetMenuBar(MenuBar)
 
+        self.canvas = Canvas(app="wx", parent=self)
+
     def on_quit(self, event):
         self.canvas.stop_timer()
         self.Close(True)
 
     def on_show(self, event):
-        self.canvas = Canvas(app="wx", parent=self, show=True)
+        self.canvas.show()
         event.Skip()
 
 

--- a/examples/tutorial/app/simple_wx.py
+++ b/examples/tutorial/app/simple_wx.py
@@ -43,14 +43,17 @@ class TestFrame(wx.Frame):
         file_menu = wx.Menu()
         file_menu.Append(wx.ID_EXIT, "&Quit")
         self.Bind(wx.EVT_MENU, self.on_quit, id=wx.ID_EXIT)
+        self.Bind(wx.EVT_SHOW, self.on_show)
         MenuBar.Append(file_menu, "&File")
         self.SetMenuBar(MenuBar)
-
-        self.canvas = Canvas(app="wx", parent=self, show=True)
 
     def on_quit(self, event):
         self.canvas.stop_timer()
         self.Close(True)
+
+    def on_show(self, event):
+        self.canvas = Canvas(app="wx", parent=self, show=True)
+        event.Skip()
 
 
 if __name__ == '__main__':

